tests/test_CRA_gan_attack.py

```diff
@@ -52,7 +52,7 @@ def set_config_femnist(self, cfg):
     def test_CRA_GAN_femnist_standalone(self):
         backup_cfg = self.set_config_femnist(global_cfg)
         setup_seed(global_cfg.seed)
-        setup_logger(global_cfg)
+        update_logger(global_cfg)
```

Comment:
For all the unit_test_xxx.py, we need use ``init_cfg = global_cfg.clone()`` rather than the ``global_cfg``. Otherwise, the recursive output directories still exist in unit test.

Reply:
> 

I will fix this in another logger PR #29